src/ekka.appup.src

```diff
@@ -1,14 +1,14 @@
 %% -*-: erlang -*-
 
-{"0.7.5",
+{"0.8.1.1",
   [
-    {"0.7.4", [
-      {load_module, ekka_locker, brutal_purge, soft_purge, []}
+    {"0.8.1", [
```

Comment:
which version is used in 4.3.0?
we should support upgrading emqx form 4.3.0 to 4.3.N (the next release).

Reply:
0.8.1
https://github.com/emqx/emqx/blob/4.3.0/rebar.config#L44